frontends/ol-search-ui/assets/learning-resource-card.scss

```diff
@@ -0,0 +1,75 @@
+$lightText: #8c8c8c !default;
+$spacer: 0.75rem !default;
+$boldWeight: 500 !default;
+$smallFontSize: 0.75em !default;
```

Comment:
I put *most* of the css for LearningResourceCard in ol-search-ui. These 4 values are customizable.

The component also accepts a class:
```ts
<LearningResourceCard resource={resource} className="your-favorite-class" />
```
so you can pass a classname to customize it more. 

I added the hover shadow by passing a class in Infinite Corridor.